contracts/Account.sol

```diff
@@ -696,6 +696,38 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         return (pendingWithdrawal.value, pendingWithdrawal.timestamp);
     }
 
+    /**
+     * @notice Votes on a proposal in the referendum stage.
+     * @param proposalId The ID of the proposal to vote on.
+     * @param index The index of the proposal ID in `dequeued`.
+     * @param yesVotes The yes votes weight.
+     * @param noVotes The no votes weight.
+     * @param abstainVotes The abstain votes weight.
+     */
+    function voteProposal(
+        uint256 proposalId,
+        uint256 index,
+        uint256 yesVotes,
+        uint256 noVotes,
+        uint256 abstainVotes
+    ) public onlyManager {
+        bool voteResult = getGovernance().votePartially(
+            proposalId,
+            index,
+            yesVotes,
+            noVotes,
+            abstainVotes
+        );
+        require(voteResult == true, "Voting was not successful");
```

Comment:
IIRC,  we choose to go with the new solidity revert Error instead of require because of readability of the former. As documented in our [contributing.md](https://github.com/celo-org/staked-celo/blob/master/CONTRIBUTING.md#errors)
this now becomes 

```
if ( voteResult != true ) { 
    revert VotingNotSuccessful(...; 
}
```